### PR TITLE
Refactoring changes

### PR DESCRIPTION
**Pull request by vihaanthora**
This PR fixes the typo in the class `HyperLocalPluginManager.java`, corrects all its references, and organizes the imports in all classes as per standard Java conventions. This pull request is built on a commit that is present in PR #176.

Tagging the mentors: @kwhetstone @arpoch

### CODE REVIEW

**Comment by MarkEWaite** [comment]
@vihaanthora this combines the changes from #176 into this pull request.  Is that intentional?  It seems to make this refactoring broader than the pull request description indicates.

**Comment by vihaanthora** [comment]
Yes @MarkEWaite , I did so because the changes in the second commit were changing the lines that were changed in the first commit. So the branch in this one is based on the branch from the previous PR. What is the ideal way to handle such a scenario?

**Comment by vihaanthora** [comment]
Should I wait till the first PR is merged before creating another to avoid merge conflicts here?

**Comment by MarkEWaite** [comment]
> Should I wait till the first PR is merged before creating another to avoid merge conflicts here?

The technique you used is just fine.  It can help reviewers if you add a note that this PR depends on #176 , though even that is optional.

**Comment by vihaanthora** [comment]
I think force-pushing after rebasing with master makes it easier to isolate changes from this PR itself since there is only one commit now. 

> It can help reviewers if you add a note that this PR depends on https://github.com/jenkins-infra/pipeline-steps-doc-generator/pull/176

I tried to imply that dependency here

> This pull request is built on a commit that is present in PR https://github.com/jenkins-infra/pipeline-steps-doc-generator/pull/176.

but I think it is better to rebase with master whenever possible.

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/HyperLocalPluginManager.java [hunk 1]
```diff
@@ -1,7 +1,37 @@
 package org.jenkinsci.pipeline_steps_doc_generator;
 
+import static hudson.init.InitMilestone.PLUGINS_LISTED;
+
+import java.io.File;
```
Comment (kwhetstone) [inline src/main/java/org/jenkinsci/pipeline_steps_doc_generator/HyperLocalPluginManager.java:5]: So was this your editor moving the java imports over the Jenkins imports?

**Review by kwhetstone — COMMENTED** [review]
It looks like the big change here is renaming the `HyperLocalPluginManager`?  The other piece is moving the Java imports over the Jenkins imports.  I looked in the main Jenkins project, and the import strategy there is all over the place, so I'm ok with this refactoring situation here.

File: src/main/java/org/jenkinsci/pipeline_steps_doc_generator/HyperLocalPluginManager.java (hunk 1 above)
Comment (vihaanthora) [reply src/main/java/org/jenkinsci/pipeline_steps_doc_generator/HyperLocalPluginManager.java:5]: Yes, this was the default behavior of auto-organize the imports.

**Comment by timja** [comment]
> I think force-pushing after rebasing with master makes it easier to isolate changes from this PR itself since there is only one commit now.

Please don't force push it makes review difficult, as soon as a review starts you should not force push anymore. you're welcome to do it before opening a PR if you want to cleanup commits from what you had locally.

**Comment by vihaanthora** [comment]
Concerned classes are now separate, these changes are reflected in `pipeline-metadata-utils`.

**Comment by vihaanthora** [comment]
Should I close this and make these changes in the PR that makes it dependent on `pipeline-metadata-utils`?

**Comment by kwhetstone** [comment]
@vihaanthora yes, let's close this and make it dependent on `pipeline-metadata-utils`

**Pull request closed without merging**